### PR TITLE
Update CGMES documentation from website version

### PR DESCRIPTION
**Pull request by colinepiloquet**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

No

**What kind of change does this PR introduce?**

Docs update

**What is the current behavior?**

A few commits were missing from the documentation that was migrated from the website to read the docs. 

**What is the new behavior (if this is a feature change)?**
CGMES documentation is up to date.

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [x] No

### CODE REVIEW

**Review by So-Fras — APPROVED** [review]

File: docs/grid_exchange_formats/cgmes/export.md [hunk 1]
```diff
@@ -2,111 +2,106 @@
...
 ## Conversion from PowSyBl grid model to CGMES
 
-The following sections describe in detail how each supported PowSyBl network model object is converted to CGMES network 
-components.
+The following sections describe in detail how each supported PowSyBl network model object is converted to CGMES network components.
 
 ### Battery
 
-PowSyBl [`Batteries`](../../grid_model/network_subnetwork.md#battery) are exported as CGMES `SynchronousMachine` with CGMES `HydroGeneratingUnits`.
+PowSyBl [`Batteries`](../model/index.md#battery) are exported as CGMES `SynchronousMachine` with CGMES `HydroGeneratingUnits`.
```
Comment (annetill) [inline docs/grid_exchange_formats/cgmes/export.md:13]: Why this link? Same remark in the whole file.

File: docs/grid_exchange_formats/cgmes/import.md [hunk 2]
```diff
@@ -385,41 +426,42 @@ This extension is used to add some CIM-CGMES characteristics to tie lines.
 
 This extension is provided by the `com.powsybl:powsybl-cgmes-extensions` module.
 
-### CGMES Tap Changers
```
Comment (annetill) [inline docs/grid_exchange_formats/cgmes/import.md:388]: Please use main version.

File: docs/grid_exchange_formats/cgmes/import.md [hunk 3]
```diff
@@ -385,41 +426,42 @@ This extension is used to add some CIM-CGMES characteristics to tie lines.
...
+### CIM-CGMES Tap Changers
```
Comment (annetill) [inline docs/grid_exchange_formats/cgmes/import.md:429]: Please use main version.

File: docs/grid_exchange_formats/cgmes/import.md [hunk 4]
```diff
@@ -385,41 +426,42 @@ This extension is used to add some CIM-CGMES characteristics to tie lines.
...
 
 <span style="color: red">TODO</span>
 
-### CGMES metadata model
```
Comment (annetill) [inline docs/grid_exchange_formats/cgmes/import.md:392]: Please use main version.

File: docs/grid_exchange_formats/cgmes/import.md [hunk 5]
```diff
@@ -385,41 +426,42 @@ This extension is used to add some CIM-CGMES characteristics to tie lines.
...
+### CIM-CGMES SSH metadata
 
 <span style="color: red">TODO</span>
 
-### CIM characteristics
+### CIM-CGMES SV metadata
 
 <span style="color: red">TODO</span>
 
-### CGMES model
```
Comment (annetill) [inline docs/grid_exchange_formats/cgmes/import.md:400]: Please use main version.

File: docs/grid_exchange_formats/cgmes/export.md (hunk 1 above)
Comment (colinepiloquet) [reply docs/grid_exchange_formats/cgmes/export.md]: Fixed

File: docs/grid_exchange_formats/cgmes/export.md [hunk 6]
```diff
@@ -24,7 +22,7 @@ PowSyBl [`BusbarSections`](../../grid_model/network_subnetwork.md#busbar-section
 
 ### DanglingLine
 
-PowSyBl [`DanglingLines`](../../grid_model/network_subnetwork.md#dangling-line) are exported as several CGMES network components.
+PowSyBl [`DanglingLines`](../../grid_model/network_subnetwork.md#dangling-line) are exported as several CGMES network component.
```
Comment (annetill) [inline docs/grid_exchange_formats/cgmes/export.md:25]: component -> objects?

**Review by annetill — APPROVED** [review]

**Pull request merged**